mig-frankfurt/dataelementhub.dal
# #16 - Remove Resourcemanager

**What's in the PR**
* remove ResourceManager.java and MigrationUtil.java
* remove flyway and slf4j dependency
* close #16 

**How to test manually**
* See PRs in [dehub-model #90](https://github.com/mig-frankfurt/dataelementhub.model/pull/90) and [dehub-rest #74](https://github.com/mig-frankfurt/dataelementhub.rest/pull/74) (links will be added when available)
